pkg/os/os.go

```diff
@@ -56,11 +58,37 @@ func OSFromEchoProcessor(output string) (*OperatingSystem, error) {
 	}
 }
 
+func OSFromSysCallUtsName(uname *syscall.Utsname) (*OperatingSystem, error) {
```

Comment:
<img width="434" alt="Screenshot 2024-03-11 at 08 57 13" src="https://github.com/daytonaio/daytona/assets/26512078/ca9732e3-567c-483c-81cb-85a4cd210ea1">

@Twacqwq, unfortunately, `syscall.Utsname` is not supported on Mac at all and the runtime panics with an exception as seen above.

I would suggest simply leaving the modification for the `fields` length and removing this `syscall.Uname` call. It's better to stick with the terminal `uname` command which is guaranteed to work.

Reply:
@Tpuljak ohh, I made an obvious mistake. You are right. I will update it immediately.